### PR TITLE
data-migration: add doc for migrate data to a downstream TiDB table with more columns

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Add one doc: migrate data to a downstream TiDB table with more columns

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/7480 (migrate-with-more-columns-downstream.md）
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: data-migration/migrate-with-more-columns-downstream.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    PRIMARY KEY (`id`)
+    )
+    ```
+
+2. Use the `operate-schema` command to set the table schema for the table to be migrated from the data source (At this time, the data migration task should be in the Paused state due to the above `Column count doesn't match` error).
+
+    {{< copyable "shell-regular" >}}
+
+    ```
+    tiup dmctl --master-addr ${advertise-addr} operate-schema set -s ${mysql-id} ${task-name} -d ${database-name} -t ${table-name} ${schema-file}
```
Comment (GMHDBJD) [inline data-migration/migrate-with-more-columns-downstream.md:73]: Prefer `source-id` rather than `mysql-id`. Please update the Chinese doc aslo.

**Review by GMHDBJD — COMMENTED** [review]
rest LGTM

File: data-migration/migrate-with-more-columns-downstream.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    ```
+
+    The descriptions of parameters in this command are as follows:
+
+    |Parameter |Description|
+    |- |-|
+    |-master-addr |-Specifies the {advertise-addr} of any DM-master node of the cluster to which dmctl is to connect. `{advertise-addr}` indicates the address that DM-master advertises to the outside world.|
```
Comment (ran-huang) [inline data-migration/migrate-with-more-columns-downstream.md:80]:
```suggestion
    |-master-addr |Specifies the `{advertise-addr}` of any DM-master node of the cluster to which dmctl is to connect. `{advertise-addr}` indicates the address that DM-master advertises to the outside world.|
```

File: data-migration/migrate-with-more-columns-downstream.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    |-operate-schema set| Manually set the schema information.|
```
Comment (ran-huang) [inline data-migration/migrate-with-more-columns-downstream.md:81]:
```suggestion
    |operate-schema set| Manually set the schema information.|
```

File: data-migration/migrate-with-more-columns-downstream.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+    |-s | Specifies the source. `{mysql-id}` indicates the source ID of MySQL data. `{task-name}` indicates the name of the synchronization task defined in the `task.yaml` configuration file of the data migration task.|
+    |-d | Specifies the database. `{database-name}` indicates the name of the upstream database. |
+    |-t |Specifies table. `{table-name}` indicates the name of the upstream data table. `{schema-file}` indicates the table schema file that to be set.|
```
Comment (ran-huang) [inline data-migration/migrate-with-more-columns-downstream.md:84]:
```suggestion
    |-t |Specifies table. `{table-name}` indicates the name of the upstream data table. `{schema-file}` indicates the table schema file that is to be set.|
```

File: data-migration/migrate-with-more-columns-downstream.md (hunk 1 above)
Comment (qiancai) [reply data-migration/migrate-with-more-columns-downstream.md]: Done in https://github.com/pingcap/docs/pull/7248/commits/7793f8d30562c2d26320ffcbb584b30aa0ab9058.

**Review by GMHDBJD — APPROVED** [review]
LGTM

**Review by ran-huang — APPROVED** [review]

**Pull request merged**